_chapters/advanced_strategy.md

```diff
@@ -16,7 +16,7 @@ Since three people are in dispute, it is more difficult to exclude them from the
 When unsure about your Chancellor choice, voting against your own candidate can provide information through the other player's votes. Where possible, have the next President (who you deem likely a Liberal) also vote for the same Chancellor. If unsure (Hitler has been enabled) get the next two Presidents to follow the tactic.
 
 ### Maximising Presidency
-If you are a possible choice for President (i.e. within two clockwise spaces of the initial president since the vote count reset), then it can often make sense to vote down governments to maximise your chance of getting the Presidency and therefore having more of an influence over the game. This is particularly important if a strong power will activated if a Fascist policy is enacted. This is a general strategy that applies to both teams as you can trust yourself more than the others to make the right decision.
+If you are a possible choice for President (i.e. within two clockwise spaces of the initial president since the vote count reset), then it can often make sense to vote down governments to maximise your chance of getting the Presidency and therefore having more of an influence over the game. This is particularly important if a strong power will activate if a Fascist policy is enacted. This is a general strategy that applies to both teams as you can trust yourself more than the others to make the right decision.
```

Comment:
From my understanding of markdown `+` and `-` just does the same thing. Is there a particular reason for switching it?